etc/standardnotes-desktop.profile

```diff
@@ -0,0 +1,29 @@
+# Firejail profile for standardnotes-desktop
+# This file is overwritten after every install/update
+# Persistent local customizations
+include /etc/firejail/standardnotes-desktop.local
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+include /etc/firejail/disable-common.inc
+include /etc/firejail/disable-passwdmgr.inc
+include /etc/firejail/disable-programs.inc
+include /etc/firejail/disable-devel.inc
+include /etc/firejail/disable-interpreters.inc
+
+whitelist ${HOME}/Standard Notes Backups
+whitelist ${HOME}/.config/Standard Notes
+
+apparmor
+caps.drop all
+netfilter
+machine-id
+nodbus
+nodvd
+nogroups
+nonewprivs
+noroot
+nosound
+notv
+protocol unix,inet,inet6,netlink
+seccom
```

Comment:
seccom -> seccomp

can the following be added below?

```
disable-mnt
private-dev
private-tmp

noexec ${HOME}
noexec /tmp
```

and could you test this private-etc?

```
private-etc ca-certificates,fonts,host.conf,hostname,hosts,resolv.conf,ssl,pki,crypto-policies,xdg
```